src/components/views/ViewReusablePaymentRequest.tsx

```diff
@@ -47,18 +82,19 @@ const ViewReusablePaymentRequest = () => {
                You currently have a Lightning Wallet set as your main account. Select an eCash mint
                as your main account to generate an eCash request.
             </p>
-         ) : fetchingPaymentRequest ? (
+         ) : paymentRequest === null ? (
             <Spinner size='xl' />
          ) : (
             <>
+               fetchingPaymentRequest
```

Comment:
if I am not wrong this will write `fetchingPaymentRequest` text to the dom

Reply:
removed